c/tskit/tables.c

```diff
@@ -578,14 +582,31 @@ tsk_individual_table_get_row(
     return ret;
 }
 
+int
+tsk_individual_table_set_metadata_schema(
+    tsk_individual_table_t *self, const char *metadata_schema)
+{
+    int ret = 0;
+    tsk_safe_free(self->metadata_schema);
+    self->metadata_schema = malloc(strlen(metadata_schema) + 1);
+    if (self->metadata_schema == NULL) {
+        ret = TSK_ERR_NO_MEMORY;
+        goto out;
+    }
+    strcpy(self->metadata_schema, metadata_schema);
+out:
+    return ret;
+}
+
 int
 tsk_individual_table_dump_text(tsk_individual_table_t *self, FILE *out)
 {
     int ret = TSK_ERR_IO;
     tsk_size_t j, k;
     tsk_size_t metadata_len;
     int err;
-
+    // TODO Feels like metadata schema should be here, but are there downstream tools
```

Comment:
Hmm, not sure. It's just for debugging really and the print_state function will output the metadata schema there. This is something we'd need to consider in the context of round-tripping a tree sequence through text I think.